movies/src/components/addMovie/AddMovie.js

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react';
+import Input from '../common/input/Input';
+import Button from '../common/button/Button';
+import Dropdown from '../common/dropdown/Dropdown';
+import Datepicker from '../common/datepicker/Datepicker';
+import CloseButton from '../closeButton/CloseButton';
+import MovieActionPopup from '../movieActionPopup/MovieActionPopup';
+import Constants from '../constants';
+
+export default function AddMovie(props) {
+  const [isMovieAction, isMovieActionPopup] = useState(false);
+  const { handleClose } = props;
+
+  const handleSubmit = (e)=> {
+    e.preventDefault();
+    isMovieActionPopup(true);
+  };
+
+  return (
+    <>
+      <CloseButton handleClose={handleClose}/>
+      <form onSubmit={handleSubmit}>
+        <p className="modal-title">Add Movie</p>
+        <Input
+          id="movie-title"
+          title="Title"
+          labelClassName="input-label"
+        />
+        <Datepicker
+          id="movie-release"
+          title="Release date"
+          labelClassName="input-label"
+          value={this}
+        />
+        <Input
+          id="movie-url"
+          title="Movie url"
+          labelClassName="input-label"
+        />
+        <div className="modal-dropdown">
+          <span className="modal-label-text">Genre</span>
+          <Dropdown options={Constants.GENRE} />
```

Comment:
Entire dropdown area shoud be clickable 
![image](https://user-images.githubusercontent.com/16895020/92574436-bfc7aa00-f28f-11ea-92be-5fd7d748e00a.png)

Reply:
Fixed